app/src/main/java/com/github/dedis/student20_pop/ui/ConnectFragment.java

```diff
@@ -77,6 +69,14 @@ private CameraSource createCamera(View view) {
                 .build();
     }
 
+
+    private void switchToCameraPermFragment() {
```

Comment:
You can rename this to switchToCameraPermissionFragment()

Reply:
Done in last commit